src/shared/ResourceForm/inputs/Number.js

```diff
@@ -1,14 +1,14 @@
 import React from 'react';
-import { FormInput } from 'fundamental-react';
+import { Input } from '@ui5/webcomponents-react';
 
 export function Number({ value = '', setValue, ...props }) {
+  if (!props.readOnly) delete props.readOnly;
```

Comment:
I don't get it, is it really necessary?

Reply:
Yup, if readOnly is present (even if it is set to false) then input is in readOnly mode